webpack/assets/javascripts/react_app/components/toast/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import Toast from './toastListitem/';
+import * as ToastActions from '../../redux/actions/toasts';
+import { connect } from 'react-redux';
+import './ToastList.scss';
```

Comment:
for index.js there is a reason, import will default to it (import './toast/') .
for index.scss it will not happen, so it is better to have a descriptive name.